app/src/main/java/it/niedermann/owncloud/notes/manageaccounts/ManageAccountsActivity.java

```diff
@@ -37,22 +38,36 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         List<LocalAccount> localAccounts = db.getAccounts();
 
         adapter = new ManageAccountAdapter((localAccount) -> SingleAccountHelper.setCurrentAccount(getApplicationContext(), localAccount.getAccountName()), (localAccount) -> {
-            db.deleteAccount(localAccount);
-            for (LocalAccount temp : localAccounts) {
-                if (temp.getId() == localAccount.getId()) {
-                    localAccounts.remove(temp);
-                    break;
+            AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            builder.setMessage("delete confirmation");
+
+            builder.setPositiveButton("confirm", (dialog, which) -> {
```

Comment:
- Also move to `strings.xml`
- Please use uppercase for the first letter of button actions
- Use "Remove" instead of "Confirm" as verb so the user is actively deciding to execute a destructive action instead of just confirming a dialog (which he will never read) :wink: